cornflow-server/cornflow/commands/permissions.py

```diff
@@ -55,6 +55,21 @@ def register_base_permissions_command(verbose):
         not in permissions_registered
     ]
 
+    for to_reg_permission in permissions_to_register:
+        if to_reg_permission in permissions_registered:
+            log.info(f"The permission to be registered {to_reg_permission} is already in the database")
+        if to_reg_permission not in permissions_registered:
+            log.info(f"The permission to be registered {to_reg_permission} is not registered. It will be registered")
+            permissions_registered.append(to_reg_permission)
```

Comment:
Why append to the ones that already are on the database? Shouldn't you append to an empty list?